doc/security/security-overview.rst

```diff
@@ -201,8 +210,8 @@ more detail; this document is part of that process.
 Execution Protection
 ====================
 
-Execution protection is planned for future releases and is roughly
-categorized into the following tasks:
+Execution protection is part of the project and can be categorized
```

Comment:
maybe just "supported"?

Reply:
that simplifies the statement, will incorporate